langchain4j-mongodb-atlas/src/test/java/dev/langchain4j/store/embedding/mongodb/MongoDbEmbeddingStoreLegacyIndexesIT.java

```diff
@@ -0,0 +1,74 @@
+package dev.langchain4j.store.embedding.mongodb;
+
+import com.mongodb.client.MongoClient;
+import com.mongodb.client.MongoCollection;
+import dev.langchain4j.data.segment.TextSegment;
+import dev.langchain4j.model.embedding.EmbeddingModel;
+import dev.langchain4j.store.embedding.EmbeddingStore;
+import dev.langchain4j.store.embedding.EmbeddingStoreIT;
+import org.bson.Document;
+import org.junit.jupiter.api.AfterEach;
+import org.testcontainers.shaded.com.google.common.collect.Sets;
+
+import java.util.Set;
+
+import static dev.langchain4j.store.embedding.mongodb.MongoDbTestFixture.EMBEDDING_MODEL;
+import static dev.langchain4j.store.embedding.mongodb.MongoDbTestFixture.createClientFromEnv;
+
+public class MongoDbEmbeddingStoreLegacyIndexesIT extends EmbeddingStoreIT {
```

Comment:
I'm curious about why this class has no `ContainerIT`?

Reply:
(Resolved, all classes now use the container through the fixture)